support-frontend/assets/components/page/pageScaffold.tsx

```diff
@@ -13,7 +13,7 @@ const container = css`
 	flex-direction: column;
 	min-height: 100vh;
 	max-width: 100%;
-	overflow-x: hidden;
+	overflow-x: clip;
```

Comment:
clip in conjunction with sticky, is how you force the sticky button to move up on the horizontal axis when the bottom of the view is dragged up. Thus you can put the sticky button inside the page scaffold now. Well done!